tests/atoms/gaia_data_layer.js

```diff
@@ -227,6 +227,7 @@ var GaiaDataLayer = {
     };
     req.onerror = function() {
       console.log('error getting setting ' + req.error.name);
+      callback('Error getting setting: ' + req.error.name);
```

Comment:
This isn't consistent with what we're doing elsewhere. In other places we return false on error, which allows us to just check for a truthy value in Python. Your solution does have the advantage of passing the details of the error, but I think we should be consistent - the error is written to the log.